tools/istio-docker.mk

```
@@ -21,7 +21,8 @@
 # It does not upload to a registry.
 docker: build-linux test-bins-linux docker.all
 
-DOCKER_TARGETS:=docker.pilot docker.proxy_debug docker.proxytproxy docker.proxyv2 docker.app docker.app_sidecar docker.test_policybackend \
+# Add new docker targets to the end of the DOCKER_TARGETS list.
+DOCKER_TARGETS:=docker.base docker.pilot docker.proxy_debug docker.proxytproxy docker.proxyv2 docker.app docker.app_sidecar docker.test_policybackend \
```

```suggestion
DOCKER_TARGETS:=docker.pilot docker.proxy_debug docker.proxytproxy docker.proxyv2 docker.app docker.app_sidecar docker.test_policybackend \
```

I think.. otherwise when you run `make docker` it will build the base, but it won't actually use it

---

I had tested this work - then I rebased on master - which had changed some logic, and didn't test...